docs/ocm-integration.md

```diff
@@ -242,3 +242,117 @@ ocm get subscriptions --parameter search="status = 'Active'"
   "trial_end_date": "0001-01-01T00:00:00Z"
 }
 ```
+
+## How to add organization capabilities to AMS
+
+AMS has the concept of _organization capabilities_ that we can use to enable
+and disable feature for certain organizations. To one of this capabilities the
+first step is to add it to the AMS code. That requires a patch similar to
+[this](https://gitlab.cee.redhat.com/service/uhc-account-manager/-/merge_requests/3168):
+
+```patch
+commit 5461cd16723bffdca00c0fe0dafd9f218bda67a5
+Author: Mat Kowalski <mko@redhat.com>
+Date:   Thu Sep 15 13:49:55 2022 +0200
+
+    MGMT-11563: Add bare metal installer multiarch capability
+
+    Contributes-to: MGMT-11563
+    Contributes-to: MGMT-11859
+    Implements-enhancement: saas-per-organization-feature-access
+
+diff --git a/pkg/api/capability_types.go b/pkg/api/capability_types.go
+index 6e909a7a..ebaee754 100644
+--- a/pkg/api/capability_types.go
++++ b/pkg/api/capability_types.go
+@@ -28,6 +28,7 @@ const CapabilitySubscribedOcpMarketplace = "capability.cluster.subscribed_ocp_ma
+ const CapabilitySubscribedOsdMarketplace = "capability.cluster.subscribed_osd_marketplace"
+ const CapabilityEnableTermsEnforcement = "capability.account.enable_terms_enforcement"
+ const CapabilityBareMetalInstallerAdmin = "capability.account.bare_metal_installer_admin"
++const CapabilityBareMetalInstallerMultiarch = "capability.account.bare_metal_installer_multiarch"
+ const CapabilityReleaseOcpClusters = "capability.cluster.release_ocp_clusters"
+ const CapabilityAutoscaleClustersDeprecated = "capability.organization.autoscale_clusters"
+ const CapabilityAutoscaleClusters = "capability.cluster.autoscale_clusters"
+@@ -60,6 +61,7 @@ var CapabilityKeys = []string{
+        CapabilitySubscribedOcpMarketplace,
+        CapabilitySubscribedOsdMarketplace,
+        CapabilityBareMetalInstallerAdmin,
++       CapabilityBareMetalInstallerMultiarch,
+        CapabilityReleaseOcpClusters,
+        CapabilityEnableTermsEnforcement,
+        CapabilityUseRosaPaidAMI,
+```
+
+That needs to be merged, and then the new version of AMS pushed to the
+corresponding environment (integration, stage and eventually production). The
+SD-B team is responsible for that, ping the `@sd-b-team` in the
+[#service-development-b](https://coreos.slack.com/archives/CBDNMS43V) channel
+in Slack.
+
+The second step is to enable the capability for the organizations that you
```

Comment:
can we enable the capability for specific email domains such as: nokia.com?

Reply:
No, the OCM accouns management system doesn't support that, capabilities need to be enabled by organization or by account.